app/templates/components/paper-dialog.hbs

```diff
@@ -5,7 +5,7 @@
     fixed=(unless parent true)
     class="md-dialog-backdrop"
     onClick=(action "outsideClicked")}}
-  {{#paper-dialog-container outsideClicked=(action "outsideClicked")}}
+  {{#paper-dialog-container class=(readonly dialogContainerClasses) outsideClicked=(action "outsideClicked")}}
```

Comment:
I named it `dialogContainerClasses` for two reasons:
 - consistency with other components (ex: `paper-select`: `class=(readonly concatenatedTriggerClasses)`)
 - avoid collision because there is multiple component we can pass class to (container, inner, ..)